integrations/parsely.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Integration: Parse.ly.
+ *
+ * @package Automattic\VIP\Integrations
+ */
+
+namespace Automattic\VIP\Integrations;
+
+/**
+ * Loads Parse.ly VIP Integration.
+ *
+ * @private
+ */
+class ParselyIntegration extends Integration {
+	/**
+	 * Loads the plugin.
+	 *
+	 * @private
+	 */
+	public function load(): void {
+		// Do not load plugin if already loaded by customer code.
+		if ( class_exists( 'Parsely\Parsely' ) ) {
+			return;
+		}
+
+		// Activating the integration via setting constant whose implementation is already
+		// handled via Automattic\VIP\WP_Parsely_Integration (ideally we should move
+		// all the implementation here such that there will be only one way of managing
+		// the plugin).
+		define( 'VIP_PARSELY_ENABLED', true );
+
+		add_filter( 'wp_parsely_credentials', array( $this, 'wp_parsely_credentials_callback' ) );
+	}
+
+	/**
+	 * Callback for `wp_parsely_credentials` filter.
+	 */
+	protected function wp_parsely_credentials_callback(): array {
+		return $this->get_config();
```

Comment:
We should only return the data that this filter needs:

- `site_id`
- `api_secret`
- `is_managed = true`

Reply:
When `activate` is triggered from platform side then we only set the site related data in `config` property not the whole integration configuration provided by VIP ([src](https://github.com/Automattic/vip-go-mu-plugins/blob/266b1587da401cef0ce8d60ae702cf630bdf446c/integrations/integrations.php#L73)) so it seems like it is already behaving  like you are suggesting.

Reply:
Done in https://github.com/Automattic/vip-go-mu-plugins/pull/4660/commits/9839b7907cc6cfd36ecb57ebcfb1ff40bf5ea928